### PR TITLE
add faq and troubleshooting to operator docs

### PR DESCRIPTION
**Pull request by shrimalmadhur**
- Added Operator Installation FAQ
- Added Operator Installation Troubleshooting

### CODE REVIEW

File: docs/operator-guides/faq.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Yes. You are required to host the metadata url publicly. The `metadata` url should always be available and should return a proper json response like [this](https://goerli-operator-metadata.s3.amazonaws.com/metadata.json)
+
+#### Am I required to publicly host logo in metadata json?
+Yes. You are required to host the logo publicly like [this](https://goerli-operator-metadata.s3.amazonaws.com/eigenlayer.png)
+
+#### Are there any restrictions to the logo image?
+Yes. We only support `.png` format and we strictly check the content of image. If your image doesn't satisfy the requirement then the EigenLayer App will not display the logo of your operator.
+
+#### What if I lose access to my keys?
+When you [create/import](https://docs.eigenlayer.xyz/operator-guides/operator-installation#create-and-list-keys) keys for the first time, it will ask a password to encrypt keys and once created, it will also show plaintext private key. Please make sure to backup the private key and the password. If you lose both you won't be able to get your keys back. If you lose the plaintext private key and still have your password you can your the export command to get your plaintext private key again.
```
Comment (QuinnLee) [inline docs/operator-guides/faq.md:21]: Can you use relative links here `https://docs.eigenlayer.xyz/operator-guides/operator-installation#create-and-list-keys` vs `/operator-installation#create-and-list-keys`

Comment (shrimalmadhur) [reply docs/operator-guides/faq.md]: resolved

**Review by wesfloyd — APPROVED** [review]
Content looks good. I can't find any issues.

File: docs/operator-guides/faq.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Yes. You are required to host the metadata url publicly. The `metadata` url should always be available and should return a proper json response like [this](https://goerli-operator-metadata.s3.amazonaws.com/metadata.json)
+
+#### Am I required to publicly host logo in metadata json?
+Yes. You are required to host the logo publicly like [this](https://goerli-operator-metadata.s3.amazonaws.com/eigenlayer.png)
+
+#### Are there any restrictions to the logo image?
+Yes. We only support `.png` format and we strictly check the content of image. If your image doesn't satisfy the requirement then the EigenLayer App will not display the logo of your operator.
+
+#### What if I lose access to my keys?
+When you [create/import](./operator-installation#create-and-list-keys) keys for the first time, it will ask a password to encrypt keys and once created, it will also show plaintext private key. Please make sure to backup the private key and the password. If you lose both you won't be able to get your keys back. If you lose the plaintext private key and still have your password you can your the export command to get your plaintext private key again.
```
Comment (joan-el) [inline docs/operator-guides/faq.md:21]: you can **run** your the export command

missing run

File: docs/operator-guides/faq.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Yes. We only support `.png` format and we strictly check the content of image. If your image doesn't satisfy the requirement then the EigenLayer App will not display the logo of your operator.
+
+#### What if I lose access to my keys?
+When you [create/import](./operator-installation#create-and-list-keys) keys for the first time, it will ask a password to encrypt keys and once created, it will also show plaintext private key. Please make sure to backup the private key and the password. If you lose both you won't be able to get your keys back. If you lose the plaintext private key and still have your password you can your the export command to get your plaintext private key again. 
+
+#### What is my operator addresss? 
+After you [create/import](./operator-installation#create-and-list-keys) ecdsa key you will be shown below log message
+```
+? Enter password to encrypt the ecdsa private key:
+ECDSA Private Key (Hex):  b3eba201405d5b5f7aaa9adf6bb734dc6c0f448ef64dd39df80ca2d92fca6d7b
```
Comment (joan-el) [inline docs/operator-guides/faq.md:27]: is this ok to display this PK?

**Review by joan-el — APPROVED** [review]
one minor edit suggestion

**Review by joan-el — APPROVED** [review]

**Pull request merged**